e2e/testrunner/runner.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 Sauce Labs Inc. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package testrunner
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os/exec"
+)
+
+type Test struct {
```

Comment:
When you use that testrunner.Test is confusing I suggest migrating all of it to Compose then it would be much simpler.